app/task/Install.php

```diff
@@ -0,0 +1,173 @@
+  +------------------------------------------------------------------------+
+*/
+
+namespace Phosphorum\Task;
+
+use Phalcon\Di;
+use League\CLImate\CLImate;
+use Phosphorum\Console\AbstractTask;
+use Phosphorum\Console\OptionParser;
+use Phosphorum\Exception\InvalidParameterException;
+
+/**
+ * Phosphorum\Task\Install
+ *
+ * @package Phosphorum\Task
+ */
+class Install extends AbstractTask
+{
+    protected $pathesList = [
+        'assets_path',
+        'log_path',
+        'pids_path',
+        'annotations_cache',
+        'config_cache',
+        'data_cache',
+        'metaData_cache',
+        'views_cache',
+        'volt_cache',
+    ];
+
+    protected $user = [
+        'userName' => 'www-data',
+    ];
+
+    /**
+     * @Doc("Check folders that must have permission to write files into them")
+     */
+    public function checkDirs()
+    {
+        if (!$this->assertNotWindows()) {
+            (new CLImate())->info("This command can't check writing permission on WINDOWS");
+        }
+
+        $this->setUserInfo($this->getCommand());
+        $this->checkPaths($this->getPaths());
+    }
+
+    /**
+     * @return bool
+     */
+    protected function assertNotWindows()
+    {
+        if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * @return array
+     */
+    protected function getCommand()
+    {
+        return OptionParser::parse($_SERVER['argv']);
+    }
+
+    /**
+     * @param array $command
+     * @return void
+     * @thrown InvalidParameterException
+     */
+    public function setUserInfo($command)
+    {
+        if (isset($command['name']) && !empty($command['name'])) {
+            $this->user['userName'] = $command['name'];
+        }
+
+        if (!$this->user = posix_getpwnam($this->user['userName'])) {
+            throw new InvalidParameterException("User '{$this->user['userName']}' hasn't been defined yet");
+        }
+    }
+
+    /**
+     * @return array
+     */
+    protected function getPaths()
+    {
+        /**@var \Phalcon\Registry*/
+        $registry = Di::getDefault()->get('registry');
+
+        $paths = [];
+        foreach ($this->pathesList as $value) {
+            $paths[] = $registry->offsetGet($value);
+        }
+
+        return $paths;
+    }
+
+    /**
+     * @return void
+     */
+    protected function checkPaths($folderList)
+    {
+        foreach ($folderList as $path) {
+            if (!$stat = $this->getStatFile($path)) {
+                continue;
+            }
+
+            $permission = fileperms($path);
+
+            /** if user is folder owner */
+            if ($stat['uid'] == $this->user['uid']) {
+                $this->outputMessage($path, $permission, -3);
+                continue;
+            }
+
+            /** if user's group id and directory's group id are equal */
+            $groupInfo = posix_getgrgid($stat['gid']);
```

Comment:
It seems to me the `posix_getgrgid` function doesn't produce meaningful results on Windows systems. Can you verify this?